RELEASE.md

```diff
@@ -50,6 +51,28 @@ Following is the Release Compatibility Matrix for PyTorch releases:
 | 1.13 | >=3.7, <=3.10 | CUDA 11.6, CUDNN 8.3.2.44 | CUDA 11.7, CUDNN 8.5.0.96 |
 | 1.12 | >=3.7, <=3.10 | CUDA 11.3, CUDNN 8.3.2.44 | CUDA 11.6, CUDNN 8.3.2.44 |
 
+## Release Cadence
+
+Following is the release cadence for year 2023/2024. Please note: these dates are subject to change, the exact dates should be announced on the [dev discuss](https://dev-discuss.pytorch.org/c/release-announcements/27) before each release.
+
+| Release | Start Date | Release Date |
+| --- | --- | --- |
+| 2.1.0 | May 2023 |  October 4, 2023 |
+| 2.1.1 | October 2023 | November 2023 |
+| 2.1.2 | November 2023 | December 2023  |
+| 2.2.0 | October 2023 | January 2024 |
+| 2.2.1 | January 2024 | February 2024 |
+| 2.2.2 | February 2024 | March 2024 |
```

Comment:
IMO table would be clearner if it followed 5 column patterns like following (it would also somewhat signify that patch releases are carried in the same branch as minor ones):
```
Minor Version | Branch open for cherry pick | Release date | First patch release date | Second patch release date|
 2.1 | Jul 2023 | Oct 2023 | Nov 2023 | Dec 2023
2.2 | Nov 2023 | Jan 2024 | Feb 2024 | Mar 2024
```

Reply:
Done